### ISSUE TITLE
Update support for mediaset.it videos

### ISSUE THREAD

**Issue by fturco** [rg3/youtube-dl#16767]
### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2018.06.18*. If it's not, read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2018.06.18**

### Before submitting an *issue* make sure you have:
- [x] At least skimmed through the [README](https://github.com/rg3/youtube-dl/blob/master/README.md), **most notably** the [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones
- [x] Checked that provided video/audio/playlist URLs (if any) are alive and playable in a browser

### What is the purpose of your *issue*?
- [ ] Bug report (encountered problems with youtube-dl)
- [x] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

### Description

I'm trying to download [this](http://www.video.mediaset.it/video/avanti_un_altro/full_2017_2018/avanti-un-altro-pure-di-sera-6-giugno_850435.html) video from mediaset.it, but `youtube-dl` only downloads a small, useless 10-seconds video instead of the full 2-hours video:

```
$ ./youtube-dl -v http://www.video.mediaset.it/video/avanti_un_altro/full_2017_2018/avanti-un-altro-pure-di-sera-6-giugno_850435.html
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', 'http://www.video.mediaset.it/video/avanti_un_altro/full_2017_2018/avanti-un-altro-pure-di-sera-6-giugno_850435.html']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2018.06.18
[debug] Python version 3.6.5 (CPython) - Linux-4.17.2-gentoo-x86_64-Intel-R-_Core-TM-2_Duo_CPU_E8400_@_3.00GHz-with-gentoo-2.6
[debug] exe versions: ffmpeg 3.4.2, ffprobe 3.4.2
[debug] Proxy map: {}
[Mediaset] 850435: Downloading video CDN JSON
[Mediaset] 850435: Downloading video info JSON
[debug] Default format spec: bestvideo+bestaudio/best
[debug] Invoking downloader on 'http://vod06.msf.ticdn.it/error/Not_Available.mp4'
[download] Destination: Avanti un altro! Pure di sera - 6 giugno-850435.mp4
[download] 100% of 549.78KiB in 00:00
```

**Comment by Vrihub** [comment]
FYI, this happens with all mediaset videos, and the "fake" video contains a message about geo-restriction.

**Comment by timendum** [comment]
This is now solved, with #16568 

Mediaset changed the site again, but youtube-dl download the old urls.
